app/src/components/Tabs/Activity/direct-measure-table.tsx

```diff
@@ -0,0 +1,201 @@
+import { ActivityValue } from "@/models/ActivityValue";
+import { EditIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Icon,
+  IconButton,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  Table,
+  Tag,
+  TagLabel,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { TFunction } from "i18next";
+import React, { FC } from "react";
+import { FiTrash2 } from "react-icons/fi";
+import { MdModeEditOutline, MdMoreVert } from "react-icons/md";
+
+interface DirectMeasureTableProps {
+  t: TFunction;
+  activityData: ActivityValue[] | undefined;
+}
+
+const DirectMeasureTable: FC<DirectMeasureTableProps> = ({
+  activityData,
+  t,
+}) => {
+  return (
+    <Box>
+      <Table variant="simple" borderWidth="1px">
+        <Thead bg="background.backgroundLight">
+          <Tr fontSize="button.sm" fontWeight="bold">
+            <Th isTruncated>{t("building-type")}</Th>
+            <Th isTruncated>{t("data-quality")}</Th>
+            <Th isNumeric isTruncated>
+              {t("co2-emissions")}
+            </Th>
+            <Th isNumeric isTruncated>
+              {t("n2o-emissions")}
+            </Th>
+            <Th isNumeric isTruncated>
+              {t("ch4-emissions")}
+            </Th>
+            <Th></Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {activityData?.map((activity: ActivityValue, i: number) => {
+            const dataQuality = activity?.dataSource.dataQuality;
+            let tagStyles = {
+              bg: "",
+              border: "",
+              color: "",
+            };
+            switch (dataQuality) {
+              case "low":
+                tagStyles = {
+                  bg: "sentiment.warningOverlay",
+                  border: "sentiment.warningDefault",
+                  color: "sentiment.warningDefault",
+                };
+                break;
+              case "medium":
+                tagStyles = {
+                  bg: "background.neutral",
+                  border: "content.link",
+                  color: "content.link",
+                };
+                break;
+
+              case "high":
+                tagStyles = {
+                  bg: "sentiment.positiveOverlay",
+                  border: "interactive.tertiary",
+                  color: "interactive.tertiary",
+                };
+                break;
```

Comment:
Might make sense to just create style variants in app-theme.ts and just referencing them by e.g. `variant={`tag-quality-${dataQuality}`}` here